### DISCUSSION TITLE
Problem with less loader in next@10.0.8

### DISCUSSION THREAD

**Discussion by binary64** [vercel/next.js#22918] [Help]
### What version of Next.js are you using?

10.0.8

### What version of Node.js are you using?

14.15.4

### What browser are you using?

cli

### What operating system are you using?

windows

### How are you deploying your application?

next dev

### Describe the Bug

```
> nx run web:serve
info  - automatically enabled Fast Refresh for 1 custom loader
info  - Using external babel configuration from C:\binary64\apps\web\.babelrc.js
error - C:/binary64/libs/ui/src/styles/antd.less 1:0
Module parse failed: Unexpected character '@' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> @import '~antd/dist/antd.less';
|
| @primary-color: #54b2d3; // primary color for all components
[ ready ] on http://localhost:4200
```

### Expected Behavior

```
> nx run web:serve
info  - automatically enabled Fast Refresh for 2 custom loaders
info  - Using external babel configuration from C:\binary64\apps\web\.babelrc.js
event - compiled successfully
[ ready ] on http://localhost:4200
```

### To Reproduce

TBD

**Comment by binary64** [comment]
Is it not a bug then? If I downgrade to 10.0.7 it works. I've upgraded many versions of nextjs, and this has only suddenly started to happen.

**Comment by hridoy43** [comment]
If you are using Antd and want less support then you can use this boilerplate. :D 

https://github.com/hridoy43/next-antd-boilerplate

**Reply by MarkLyck** [reply]
boilerplate uses next@10.

Looking for a next@11 version.
